src/pymovements/gaze/io.py

```diff
@@ -332,7 +332,7 @@ def from_asc(
     │ 2339290 ┆ 618.0 ┆ [637.6, 531.4] │
     │ 2339291 ┆ 618.0 ┆ [637.3, 531.2] │
     └─────────┴───────┴────────────────┘
-    >>> metadata['sampling_rate']
+    >>> float(metadata["recording_config"][0]['sampling_rate'])
```

Comment:
this line should be the same as the one in the main branch:
https://github.com/aeye-lab/pymovements/blob/fd01f61a46793ebf9090042060be8d0ba0316b33/src/pymovements/gaze/io.py#L342

Reply:
Ah I see this was already resolved by my merge commit